src/Nancy/GlobalizationConfigurationExtensions.cs

```diff
@@ -13,10 +14,18 @@ public static class GlobalizationConfigurationExtensions
         /// </summary>
         /// <param name="environment">An <see cref="INancyEnvironment"/> that should be configured.</param>
         /// <param name="supportedCultureNames">Cultures that the application can accept</param>
-        public static void Cultures(this INancyEnvironment environment, IEnumerable<string> supportedCultureNames)
+        /// <param name="defaultCulture">Used to set a default culture for the application</param>
+        /// <remarks>If defaultCulture not specified the first supported culture is used</remarks>
+        public static void Cultures(this INancyEnvironment environment, IEnumerable<string> supportedCultureNames, string defaultCulture = null)
         {
+            if (supportedCultureNames != null && supportedCultureNames.Any() && defaultCulture == null)
+            {
+                defaultCulture = supportedCultureNames.FirstOrDefault();
+            }
+

```

Comment:
This is where we should make sure that `defaultCulture` actually is in the list of `supportedCultureNames`